CHANGELOG.md

```diff
@@ -4,6 +4,73 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v5.0.0](https://github.com/buildkite/elastic-ci-stack-for-aws/compare/v4.5.0...master) (DRAFT)
+
+### Added
+<!-- 
+    TODO: document the new lambda scaler properly as a new default and what it means. For now https://forum.buildkite.community/t/experimental-lambda-based-scaler/425 is a good starting point.
+-->
+* **Our blazing fast lambda scaler is now the default** which makes for much faster scaling in response to pending jobs [#575](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/575) (@lox)
+* **Windows support** on a new Windows Server 2019 based image [#546](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/546) ([jeremiahsnapp](https://github.com/jeremiahsnapp))
```

Comment:
Maybe we should mark windows support as Experimental?

There's a few folks using it effectively already, but it's definitely not as battle tested as the linux support and I expect we'll get some great feedback as more customers try it.